doc/cascadia/profiles.schema.json

```diff
@@ -931,6 +931,10 @@
           "description": "When set to true, enable retro terminal effects. This is an experimental feature, and its continued existence is not guaranteed.",
           "type": "boolean"
         },
+        "experimental.pixelShaderEffect": {
+          "description": "If set to a file name or preset, compiles the pixel shader effect and applies it the terminal view. Currently 2 presets are supported `RETRO` and `RETROII`. This is an experimental feature, and its continued existence is not guaranteed.",
```

Comment:
I think this description needs updating. I don't see RETROII anymore.

Reply:
sure does